knox-auto-policy-chart/values.yaml

```diff
@@ -95,75 +95,72 @@ tolerations: []
 
 affinity: {}
 
+configmapAutoPolicyDiscovery:
+  enabled: true
+  conf: "configmapfiles/conf.yaml"
+
+
 # conf.yaml
-config:
-  application:
-    name: knoxautopolicy
-    network:
-      operation-mode: 1                         # 1: cronjob | 2: one-time-job
-      cron-job-time-interval: "0h0m10s"         # format: XhYmZs 
-      operation-trigger: 10000
-      network-log-limit: 10000
-      network-log-from: "db"                    # db|hubble
-      network-log-file: "./flow.json"           # file path
-      network-policy-to: "db"              # db, file
-      network-policy-dir: "./"
-      network-policy-types: 3
-      network-policy-rule-types: 511
-    system:
-      operation-mode: 1                         # 1: cronjob | 2: one-time-job
-      cron-job-time-interval: "0h0m10s"         # format: XhYmZs 
-      operation-trigger: 1000
-      system-log-limit: 10000
-      system-log-from: "db"                     # db|kubearmor
-      system-log-file: "./log.json"             # file path
-      system-policy-to: "db"               # db, file
-      system-policy-dir: "./"
-      deprecate-old-mode: true
-    cluster:
-      cluster-info-from: "accuknox"            # k8sclient|accuknox
-      cluster-mgmt-url: "http://cluster-management-service.accuknox-dev-cluster-mgmt.svc.cluster.local"
-
-  database:
-    driver: mysql
-    host: accuknox-dev-mysql-haproxy.accuknox-dev-mysql-db.svc.cluster.local
-    port: 3306
-    user: accuknox_user
-    password: EAy5Kq4uhc5Gkws
-    dbname: accuknox
-    table-network-log: network_log
-    table-network-policy: network_policy
-    table-system-log: system_log
-    table-system-alert: system_alert
-    table-system-policy: system_policy
-
-  feed-consumer:
-    kafka:
-      number-of-consumers: 3
-      broker-address-family: v4
-      session-timeout-ms: 6000
-      auto-offset-reset: "earliest"
-      bootstrap-servers: "dev-kafka-kafka-external-bootstrap.accuknox-dev-kafka.svc.cluster.local:9095"
-      group-id: knoxautopolicy
-      topics: 
-        - cilium-telemetry-new
-        - cilium-telemetry
-        - kubearmor-syslogs
-      security:
-        protocol: SSL
-      ssl:
-        enabled: true
-      ca:
-        location: /kafka-ssl/ca.pem
-      keystore:
-        location: /kafka-ssl/user.p12
-        pword: DHmHwNYy22At
-      events:
-        buffer: 50
-
-  logging:
-    level: "INFO"
-    
-  cilium-hubble:
-    url: 10.4.41.240
-    port: 80
+# config:
```

Comment:
can we delete these commented configs?

Reply:
okay

Reply:
@nyrahul  I have made changes, please review it